paddle/fluid/pybind/eager_math_op_patch.cc

```diff
@@ -228,17 +232,66 @@ static PyObject* tensor__add__method(TensorObject* self,
       ret = CallScalarFuction(self_tensor, other, "add");
```

Comment:
shall we overload `CallScalarFuction` when other is type of int or bool

Reply:
Will fix with comment above.